ocs_ci/ocs/utils.py

```diff
@@ -600,14 +600,19 @@ def create_oc_resource(
     occli.create(cfg_file)
 
 
-def get_pod_name_by_pattern(pattern='client', namespace='openshift-storage'):
+def get_pod_name_by_pattern(
+    pattern='client',
+    namespace='openshift-storage',
```

Comment:
I know it's not your definition here but can we change it to 
```python
namespace=None,
```

And in the function have
```python
namespace = namespace if namespace else config.ENV_DATA['cluster_namespace']
```